src/System.Net.Requests/src/project.json

```diff
@@ -8,6 +8,7 @@
         "System.Diagnostics.Tracing": "4.0.0",
         "System.IO": "4.0.10",
         "System.IO.Compression": "4.1.0",
+        "System.Net.Http": "4.0.0",
```

Comment:
This was already there in 1.1, and i think this is correct, we want to always reference the lowest possible reference version a lib can support, meaning this implementation will work with System.Net.Http reference libs >= 4.0.0

If this implementation of System.Net.Requests requires newer APIs from System.Net.Http, only then will we need a higher version of 4.1.0, but I don't think that's the case here, else build will fail, complaining it can't find those types in 4.0.0 reference lib.

cc @ericstj